src/js/sjcl-custom/sjcl-ecdsa-signdeterministic.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+/*global sjcl*/
+/*eslint-disable new-cap*/
+
+function generateK(secretKey, hash, hashObject) {
```

Comment:
Definitely not. Another reference: https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/ecdsa.js#L11-L12